tests/setup.sh

```diff
@@ -12,8 +12,19 @@ elif [ "$SUITE" = "phpunit-full" ];
 then
     export MOODLE_DOCKER_PHPUNIT_EXTERNAL_SERVICES=true
     initcmd="bin/moodle-docker-compose exec -T webserver php admin/tool/phpunit/cli/init.php"
-elif [ "$SUITE" = "app" ];
+elif [ "$SUITE" = "behat-app" ];
 then
+    git clone --branch v3.8.2 --depth 1 git://github.com/moodlehq/moodle-local_moodlemobileapp $HOME/moodle/local/moodlemobileapp
```

Comment:
Shouldn't this be using $APP_VERSION instead of harcoded `v3.8.2`?

Reply:
Yes that's a good improvement, I've added it :).

Reply:
Good!